### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by cvanderaa**
Hi Marcel @LiNk-NY ,

This is a very small PR to change 2 things:

- The assays in the `macrophage_differentiation` project (SCoPE2) were called `macrophage_protein` and `macrophage_rna`. Laurent pointed out to me that this can be confusing because monocyte data are also present in the assays. I renamed the assays using the project abbreviation `macroph_diff_protein` and `macroph_diff_rna`.
- I adapted the vignette with the new assay names and fixed some typos.

### CODE REVIEW

**Comment by LiNk-NY** [comment]
Thanks for the PR Chris, note that our new function `SingleCellMultiModal` will return an assay with a name like `macrophage_differentiation_macroph_diff_protein`, for example.

**Comment by cvanderaa** [comment]
Oh sorry, I didn't see the new function. It's a great feature you implemented there! 

So, I could simply name the assays `rna` and `protein`, and once called from `SingleCellMultiModal(...)`, those will become `macrophage_differentiation_rna` and `macrophage_differentiation_protein`?

Should I also document this new way of retrieving multiple datasets in the `SCoPE2.Rmd` vignette?

**Comment by LiNk-NY** [comment]
Hi Chris, @cvanderaa 

> So, I could simply name the assays `rna` and `protein`, and once called from `SingleCellMultiModal(...)`, those will become `macrophage_differentiation_rna` and `macrophage_differentiation_protein`?
 
Yes, those would be the names you get with the new function. I had to append the `DataType` to differentiate the different technologies in the assays. I should add a note about that in the documentation. 

> Should I also document this new way of retrieving multiple datasets in the `SCoPE2.Rmd` vignette?

You could but you don't have to. The function is complementary to the existing ones.

**Comment by cvanderaa** [comment]
Hi Marcel @LiNk-NY ,

Thanks for the info! So, this PR simply renames the assays to `rna` and `protein` (= names of the modalities on `ExperimentHub`) and makes some slight improvements to the SCoPE2 vignette.

File: R/SCoPE2.R [hunk 1]
```diff
@@ -87,7 +87,7 @@ SCoPE2 <- function(DataType = "macrophage_differentiation",
 
     ## Rename assays with project
     names(ess_list[["experiments"]]) <-
```
Comment (LiNk-NY) [inline R/SCoPE2.R]: It looks like you don't need this line anymore? @cvanderaa

**Review by LiNk-NY — CHANGES_REQUESTED** [review]
This looks neat! If you could just remove that line that is not doing anything, we'll be good to go. Thanks!

**Comment by cvanderaa** [comment]
Oops, sorry about that! I have removed it!

**Review by LiNk-NY — APPROVED** [review]

**Pull request merged**